components/bio-formats-tools/src/loci/formats/tools/ImageConverter.java

```diff
@@ -196,7 +196,7 @@ else if (args[i].equals("-tiley")) {
           catch (NumberFormatException e) { }
         }
         else if (!args[i].equals(CommandLineTools.NO_UPGRADE_CHECK)) {
-          LOGGER.error("Found unknown command flag: {}; exiting.", args[i]);
+          System.out.println("Found unknown command flag: " + args[i] + "; exiting.");
```

Comment:
For me the new test is able to run successfully and check the output from LOGGER without the need to us System.out directly. Can this be reverted or is there a reason its needed here?